Index: scripts/release-notes.sh
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- scripts/release-notes.sh	(revision )
+++ scripts/release-notes.sh	(revision )
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+# To install:
+# npm install -g git-release-notes
+
+./node_modules/.bin/git-release-notes v`npm view . version`..master ./scripts/release-notes.ejs
+#./node_modules/.bin/git-release-notes v`npm view . version`..master ./scripts/release-notes.ejs >> ./commits.md
+#echo "# `npm view . version`" > tempHistory.md
+#cat ./commits.md >> ./tempHistory.md
+#cat ./HISTORY.md >> ./tempHistory.md
+#cat ./tempHistory.md > ./HISTORY.md
+#rm ./commits.md
+#rm ./tempHistory.md
Index: scripts/prepend-history.sh
===================================================================
--- scripts/prepend-history.sh	(revision d180b039deb2a3bbb47eea4b901c84516dac59a9)
+++ scripts/prepend-history.sh	(revision d180b039deb2a3bbb47eea4b901c84516dac59a9)
@@ -1,10 +0,0 @@
-#!/usr/bin/env bash
-
-# To install:
-# npm install -g git-release-notes
-
-echo "# vNext" > tempHistory.md
-git-release-notes $1..master ./scripts/release-notes.ejs >> ./tempHistory.md
-cat ./HISTORY.md >> ./tempHistory.md
-cat ./tempHistory.md > ./HISTORY.md
-rm ./tempHistory.md
